test/lib/subgraphPoolDataService.ts

```diff
@@ -43,6 +43,9 @@ const queryWithLinear = `
           c
           s
           lambda
+          delta
+          epsilon
+          fxRate
```

Comment:
The Subgraph doesn't seem to have `fxRate` field and throws an error with this query?

Reply:
i was not aware of the change of the name while we are developing this. changed the name to `latestFXPrice`

it's appearing when i query:
```
{
 tokens (where : {symbol: "XSGD"}){
     symbol
     latestFXPrice
  }
}
```

hope this works :)

Reply:
Unfortunately we can't query like that as we need to fetch all the pools/tokens for the SOR. Also it seems like you have added `latestFxPrice` to the main pools query in subgraphPoolDataService but that throws an error. Maybe you need to add something to the Subgraph?